dist/images/Makefile

```diff
@@ -51,5 +51,5 @@ daemonsetyaml:
 BRANCH = $(shell git rev-parse  --symbolic-full-name HEAD)
 COMMIT = $(shell git rev-parse  HEAD)
 bld: ../../go-controller/_output/go/bin/ovnkube
-	cp ../../go-controller/_output/go/bin/* .
+	cp -r ../../go-controller/_output/go/bin/* .
```

Comment:
@trozet why `-r` here? are there any folders under `_output/go/bin` that we generate now?

Reply:
oh, that was because when the windows binaries were added they are added to a ./windows dir. The CI never caught it because before it only tried to build the images if the Dockerfile changed.

Reply:
@trozet understood. Makes sense.